requirements.txt

```diff
@@ -1,48 +1,49 @@
 -e git+https://github.com/OpenDataServices/flatten-tool.git@4c13ef0b32a59e810919a3de09bc8f64ce8f9392#egg=flattentool
--e git+https://github.com/open-contracting/lib-cove-ocds.git@9cf68780f9e30921e432b12e3e346a057e2bbba5#egg=libcoveocds
+-e git+https://github.com/OpenDataServices/lib-cove.git@52acb2f0dcd296b9d952ff2adbae61dc4f8dd296#egg=libcove
+-e git+https://github.com/open-contracting/lib-cove-ocds.git@8a3d696eb614063bdec1790c0e3fda68d0445040#egg=libcoveocds
 ocdskit==0.0.5
 Flask==1.0.2
-alembic==1.0.7
+alembic==1.0.9
 pgpasslib==1.1.0
-psycopg2==2.7.7
+psycopg2==2.8.2
 ocdsmerge==0.5.1
 blinker==1.4
-redis==3.2.0
-sentry-sdk==0.7.4
+redis==3.2.1
+sentry-sdk==0.7.10
+# Lock to 1.2 branch (1.3 branch has issues with an identifier being too long)
+SQLAlchemy==1.2.19
 ## The following requirements were added by pip freeze:
-attrs==18.2.0
 bleach==3.1.0
 cached-property==1.5.1
-certifi==2018.11.29
+certifi==2019.3.9
 chardet==3.0.4
 Click==7.0
 commonmark==0.8.1
 contextlib2==0.5.5
-Django==2.1.7
+Django==2.2
 et-xmlfile==1.0.1
 future==0.17.1
 idna==2.8
 itsdangerous==1.1.0
 jdcal==1.4
-Jinja2==2.10
+Jinja2==2.10.1
 json-merge-patch==0.2
 jsonref==0.2
-jsonschema==3.0.1
-lxml==4.3.2
-Mako==1.0.7
+jsonschema==2.6.0
```

Comment:
why going back to an earlier version?

Reply:
Nothing I've done.

There was a problem with the new release and we locked it in a lib https://github.com/OpenDataServices/lib-cove/blob/master/setup.py#L14

Reply:
seems legit :)